internal/venice-consumer/src/main/java/com/linkedin/venice/pubsub/kafka/KafkaMessage.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.venice.pubsub.kafka;
+
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+
+public class KafkaMessage<K, V> implements PubSubMessage<K, V, KafkaTopicPartition, Long> {
```

Comment:
IMO,` PubSubMessage` should be a class instead of the interface. To create an object of PSM we can pass the specific message as a parameter to the PSM constructor. 
Ex.
`PubSubMessage msg = new PubSubMessage(ConsumerRecord record)`;

Reply:
Passing a specific message (e.x `ConsumerRecord` is from Kafka world), this will make the `PubSubMessage` less general by coupling too much with kafka.

Reply:
I don't think this is right.